frontend/src/scenes/project/Settings/CorrelationConfig.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { useActions, useValues } from 'kea'
+import { teamLogic } from 'scenes/teamLogic'
+import { personPropertiesModel } from '~/models/personPropertiesModel'
+import { Select } from 'antd'
+import { DEFAULT_EXCLUDED_PERSON_PROPERTIES } from 'scenes/funnels/funnelLogic'
+
+export function CorrelationConfig(): JSX.Element {
+    const { updateCurrentTeam } = useActions(teamLogic)
+    const { currentTeam } = useValues(teamLogic)
+    const { personProperties } = useValues(personPropertiesModel)
+
+    const handleChange = (excludedProperties: string[]): void => {
+        updateCurrentTeam({ correlation_config: { excluded_person_property_names: excludedProperties } })
+    }
+
+    return (
+        <div style={{ marginBottom: 16 }}>
+            <div style={{ marginBottom: 8 }}>
+                {currentTeam && (
+                    <>
+                        Excluded person properties:{' '}
+                        <Select
```

Comment:
I think we can reuse: PropertyFilters here https://github.com/PostHog/posthog/blob/master/frontend/src/scenes/project/Settings/TestAccountFiltersConfig.tsx#L22

By passing in a TaxonomicGroupType, it handles loading persons and loading events for you

Reply:
oh, maybe not PropertyFilters. Let me look for regular filters if there are any 😅

Reply:
I'm thinking, can we make this a followup? To avoid any more potential conflicts. It is, after all, behind a feature flag still.

Reply:
Yeah 100%

Reply:
I have had a look at https://github.com/PostHog/posthog/blob/use-taxo-filter/frontend/src/lib/components/TaxonomicFilter/TaxonomicFilter.tsx#L15:L15

I assume actually we would want to do something like `PropertyFilters` but without the values @neilkakkar ?

Reply:
Precisely. An existing example of this exact functionality is this: https://github.com/PostHog/posthog/blob/master/frontend/src/scenes/insights/BreakdownFilter/TaxonomicBreakdownFilter.tsx#L20

For Cohorts breakdown on funnels, we allow choosing more than one, and all of those get added to the list.